docs/persistence.rst

```diff
@@ -0,0 +1,84 @@
+.. _persistence:
+
+Secure persistence with skops
+=============================
+
+.. warning::
+
+   This feature is very early in development, which means the API is
+   unstable and it is **not secure** at the moment. Therefore, use the same
+   caution as you would for ``pickle``: Don't load from sources that you
+   don't trust. In the future, more security will be added.
+
+Skops offers a way to save and load sklearn models without using :mod:`pickle`.
+The ``pickle`` module is not secure, but with skops, you can securely save and
+load sklearn models without using ``pickle``.
+
+``Pickle`` is the standard serialization format for sklearn and for Python in
+general. One of the main advantages of ``pickle`` is that it can be used for
+almost all Python code but this flexibility also makes it inherently insecure.
+This is because loading certain types of objects requires the ability to run
+arbitrary code, which can be misused for malicious purposes. For example, an
+attacker can use it to steal secrets from your machine or install a virus. As
+the Python docs say:
```

Comment:
I was thinking something like this would be enough?

```suggestion
attacker can use it to steal secrets from your machine or install a virus. As
the `Python docs <https://docs.python.org/3/library/pickle.html#module-pickle>`__ say:
```

This whole paragraph doesn't have any links to pickle.

Reply:
This results in the exact same link as ```:mod:`pickle` ``` in the paragraph directly above, but anyway, I added it